src/cloud/accesskeys.ts

```diff
@@ -25,7 +25,7 @@ export function getAccessKeys(profile: Profile, filters?: FiltersAccessKeys): Pr
         }
         return res.accessKeys;
     }, (err_: any) => {
-        return "Error, bad credential or region?" + err_;
+        return err_;
```

Comment:
just a comment, not blocker: "_" is used to mean non-used variables, maybe you can just name it "err"